docs/classic-ui/forms.md

```diff
@@ -120,6 +120,324 @@ schema = IMyForm
+           name="not_last"
+           form:before="*">
+        <title>Not last</title>
+    </field>
+```
+
+
+### Organizing fields into fieldsets
+
+Fields can be grouped into fieldsets, which will be rendered within an HTML
+`<fieldset>` tag. In this example the `footer` and `dummy` fields
+are placed within the `extra` fieldset:
+
+```python
+    from plone.supermodel import model
+    from plone.autoform import directives as form
+
+    class IMySchema(model.Schema):
+
+        model.fieldset('extra',
+            label=u"Extra info",
+            fields=['footer', 'dummy']
+            )
+
+        footer = schema.Text(
+            title=u"Footer text",
+            )
+
+        dummy = schema.Text(
+            title=u"Dummy"
+            )
+```
+
+In supermodel XML fieldsets are specified by grouping fields within a
+`<fieldset>` tag:
+
+```xml
+  <fieldset name="extra" label="Extra info">
+      <field name="footer" type="zope.schema.TextLine">
+          <title>Footer text</title>
+      </field>
+      <field name="dummy" type="zope.schema.TextLine">
+          <title>Dummy</title>
+      </field>
+  </fieldset>
+```
+
+
+### Changing a field's widget
+
+Usually, z3c.form picks a widget based on the type of your field.
+You can change the widget using the `widget` directive if you want
+users to enter or view data in a different format. For example,
+here we change the widget for the `human` field to use yes/no
+radio buttons instead of a checkbox:
+
+```python
+    from plone.supermodel import model
+    from plone.autoform import directives as form
+    from z3c.form.browser.radio import RadioFieldWidget
+
+    class IMySchema(model.Schema):
+        form.widget('human', RadioFieldWidget)
+        human = schema.Bool(
+            title = u'Are you human?',
+            )
+```
+
+You can also pass widget parameters to control attributes of the
+widget. For example, here we keep the default widget, but
+set a CSS class:
+
+```python
+    from plone.supermodel import model
+    from plone.autoform import directives as form
+    from z3c.form.browser.radio import RadioWidget
+
+    class IMySchema(model.Schema):
+        form.widget('human', klass='annoying')
+        human = schema.Bool(
+            title = u'Are you human?',
+            )
+```
+
+In supermodel XML the widget is specified using a `<form:widget>` tag, which
+can have its own elements specifying parameters::
+
+```xml
+    <field name="human" type="zope.schema.TextLine">
+        <title>Are you human?</title>
+        <form:widget type="z3c.form.browser.radio.RadioWidget">
+            <klass>annoying</klass>
+        </form:widget>
+    </field>
+```
+
+Note: In order to be included in the XML representation of a schema,
+widget parameters must be handled by a WidgetExportImportHandler utility.
+There is a default one which handles the attributes defined in
+`z3c.form.browser.interfaces.IHTMLFormElement`.
+
+### Protect a field with a permission
```

Comment:
chapter: fields feels more like the right place for this with a reference from here maybe.